### DISCUSSION TITLE
开始菜单汉化问题

### DISCUSSION THREAD

**Discussion by KIITTID** [zetaloop/ExplorerPatcher#29] [General]
通过外国网友分享的方法修复了WIN10开始菜单模式下UWP应用不显示的问题：https://github.com/valinet/ExplorerPatcher/issues/1591#issuecomment-1635373375。
但是发现以前使用epwin10startCN.zip文件替换完善开始菜单汉化的办法不行了，出现了很多新的英文单词未完全汉化，希望大佬能出个新的完整的汉化文件呀。

**Comment by zetaloop** [comment]
我试试这个方案，行的话直接整个做到新的开始菜单补充包里。
你试了新的补充包2.1了吗？https://github.com/zetaloop/ExplorerPatcher

**Reply by KIITTID** [reply]
我实践了，确实可以完全汉化，谢谢开发者。👍👍👍

**Comment by zetaloop** [comment]
已经把这个功能放到补充包里了，见 https://github.com/zetaloop/ExplorerPatcher

**Reply by KIITTID** [reply]
太棒了
